R/pkg/R/sparkR.R

```diff
@@ -369,6 +372,24 @@ sparkR.session <- function(
     overrideEnvs(sparkConfigMap, paramMap)
   }
```

Comment:
I'm a bit confused. To be concrete I think we were talking about something like:
```
    sparkConfigMap <- convertNamedListToEnv(sparkConfig)
    namedParams <- list(...)
    if (length(namedParams) > 0) {
      paramMap <- convertNamedListToEnv(namedParams)
     # Override for certain named parameters
     if (exists("spark.master", envir = paramMap)) {
       master <- paramMap[["spark.master"]]
     }
     if (exists("spark.app.name", envir = paramMap)) {
       appName <- paramMap[["spark.app.name"]]
     }
      overrideEnvs(sparkConfigMap, paramMap)
    }
   if (nzchar(master)) {
     sparkConfigMap[["spark.master"]] <- master
   }
```